packages/fields/src/types/Color/views/Field.js

```diff
@@ -12,10 +10,23 @@ const ColorField = ({ field, value: serverValue, errors, onChange }) => {
   const value = serverValue || '';
   const htmlID = `ks-input-${field.path}`;
 
+  const colorPickerValue = useMemo(() => {
+    // keystone previously stored values as a hex string and this should still be supported
+    // it is now stored as an rgba string
+    if (value) {
+      if (value.indexOf('rgba', 0) === 0) {
+        const rgbaValues = value.replace(/^rgba?\(|\s+|\)$/g, '').split(',');
```

Comment:
```suggestion
        const rgbaValues = value.replace(/^rgba\(|\s+|\)$/g, '').split(',');
```
The `a` is required based on the `if` check above.